### PR TITLE
Web api sprint 1

### PR DESCRIPTION
**Pull request by mardiel**
Updated to use dotnet core 2.2, and postgres 9.6 - the docker-compose should be working too.

### CODE REVIEW

File: prime-dotnet-webapi/ApiDbContext.cs [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                .IsUnique(false);
+
+            modelBuilder.Entity<Address>()
+                .HasIndex("EnrolleeId", "AddressType")
+                .HasName("IX_EnrolleeId_AddressType")
+                .IsUnique(true);
+            #endregion
+
+            #region Relationships
+            // modelBuilder.Entity<MailingAddress>(entity =>
```
Comment (itchison) [inline prime-dotnet-webapi/ApiDbContext.cs]: Is there a reason for the commented out code here?

File: prime-dotnet-webapi/Controllers/EnrolmentController.cs [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Models;
+using Prime.Services;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/v1/[controller]")]
```
Comment (itchison) [inline prime-dotnet-webapi/Controllers/EnrolmentController.cs]: This is fine for now, but we should have a bigger conversation about api and version in the pathing

File: prime-dotnet-webapi/ApiDbContext.cs (hunk 1 above)
Comment (mardiel) [reply prime-dotnet-webapi/ApiDbContext.cs]: This was accidentally left in the commit - I have removed it now.

File: prime-dotnet-webapi/Controllers/EnrolmentController.cs (hunk 2 above)
Comment (mardiel) [reply prime-dotnet-webapi/Controllers/EnrolmentController.cs]: After discussion the API versioning will be handled by the routing in OpenShift, so it does not need to be in the code.  From a dotnet routing perspective, there will be no version in the endpoint URL.

**Pull request merged**